ProcessMaker/ScreenConsolidator.php

```diff
@@ -119,7 +119,9 @@ private function setNestedScreen($item, &$new, $index0 = 0)
     private function setRecordList($item, &$new, $index0 = 0)
     {
         $pageId = $item['config']['form'];
-        $item['config']['form'] = $pageId + $index0;
+        if ($pageId)  {
```

Comment:
Please keep the string type when solving the PR:
https://github.com/ProcessMaker/processmaker/pull/3613/files